hackclub.com.yaml

```diff
@@ -1,4327 +1,4485 @@
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 horizon:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 hsi:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 haunted:
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 hauntedhouse:
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 hauntedto:
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 hub.zephyr:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 ia:
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 ice:
-  - type: CNAME
-    value: hackclub.github.io.
+- ttl: 300
+  type: CNAME
+  value: hackclub.github.io.
 identity:
-  - ttl: 600
-    type: CNAME
-    value: a.limited.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.limited.selfhosted.hackclub.com.
 icons:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 id:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 iframe:
- - ttl: 600
-   type: CNAME
-   value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 id.assemble:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 illumination:
-  - ttl: 600
-    type: CNAME
-    value: 0b4e7cc575f0aff2.vercel-dns-017.com.
+- ttl: 300
+  type: CNAME
+  value: 0b4e7cc575f0aff2.vercel-dns-017.com.
 imperial:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: ihs-hacklab.github.io.
 improving:
-    ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 infill:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 innkeeper:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: computational-wildfowl-4zkl40wjjsepa2w7iivn026m.herokudns.com.
 intranet:
-  - ttl: 600
-    type: CNAME
-    value: intranet-72n.pages.dev.
+- ttl: 300
+  type: CNAME
+  value: intranet-72n.pages.dev.
 intro.moonbeam:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 irvine:
-  - ttl: 3600
-    type: TXT
-    values:
-      - v=spf1 include:_spf.google.com ~all
-      - google-site-verification=n9knjT30SgPd80zLKBD6M8fPFGIQUAO5OgfjDsPSrRM
-  - ttl: 3600
-    type: MX
-    values:
-      - exchange: aspmx.l.google.com.
-        preference: 1
-      - exchange: alt1.aspmx.l.google.com.
-        preference: 5
-      - exchange: alt2.aspmx.l.google.com.
-        preference: 5
-      - exchange: alt3.aspmx.l.google.com.
-        preference: 10
-      - exchange: alt4.aspmx.l.google.com.
-        preference: 10
-  - ttl: 600
-    type: ALIAS
-    value: cname.vercel-dns.com.
```

Comment:
This MX record should not have been deleted! This is a breaking change, was this communicated to the team responsible?